ReachCare/core/views.py

```diff
@@ -112,14 +114,18 @@ def get_response_message(user_questionnaire):
         return ZIP_CODE_QUESTION
 
     if user_questionnaire.zip_code is not None:
+        if user_questionnaire.next_response is not None:
+            user_questionnaire.result_index = user_questionnaire.result_index + 1
```

Comment:
Won't this increment on any message once a user sends "next"? I think we might want to do the increment in the "process_response" (i.e. only increment when the text is actually "next")

Reply:
Maybe we want something like:
```
if parse_next(..):
   self.result_index += 1
```
then I think we don't need the "next_response" field.

Reply:
Good idea. I will change it tonight and commit the changes.